http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtAsyncHttpClient.java

```diff
@@ -105,13 +106,25 @@ private AwsCrtAsyncHttpClient(DefaultBuilder builder, AttributeMap config) {
 
             this.initialWindowSize = builder.initialWindowSize;
             this.maxConnectionsPerEndpoint = maxConns;
-            this.monitoringOptions = builder.monitoringOptions;
+            this.monitoringOptions = revolveHttpMonitoringOptions(builder.connectionHealthChecksConfiguration);
             this.maxConnectionIdleInMilliseconds = config.get(SdkHttpConfigurationOption.CONNECTION_MAX_IDLE_TIMEOUT).toMillis();
 
             this.proxyOptions = buildProxyOptions(builder.proxyConfiguration);
         }
     }
 
+    private HttpMonitoringOptions revolveHttpMonitoringOptions(ConnectionHealthChecksConfiguration config) {
+        if (config == null) {
+            return null;
+        }
+
+        HttpMonitoringOptions httpMonitoringOptions = new HttpMonitoringOptions();
+        httpMonitoringOptions.setMinThroughputBytesPerSecond(config.minThroughputInBytesPerSecond());
+        int seconds = (int) (TimeUnit.MILLISECONDS.toSeconds(config.allowableThroughputFailureInterval().toMillis()));
```

Comment:
`Duration` has `getSeconds()`

Reply:
`getSeconds` only get the seconds from the duration, i.e., not the total number of whole seconds in the duration

`toSeconds` is added since 9,, so we can't use it unfortunately. :(
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/Duration.html#toSeconds()

Reply:
http://hg.openjdk.java.net/jdk9/jdk9/jdk/file/7f644a5d554a/src/java.base/share/classes/java/time/Duration.java they seem to have the same impl :P

Reply:
don't feel strongly about this though

Reply:
Interesting! Didn't know they have the same impl. Why do they have two APIs then?! 🤔

Reply:
Found this relevant link: https://bugs.openjdk.java.net/browse/JDK-8142936 
TL;DR They added `toSeconds` to be consistent with other `toXXX` methods